src/components/touchables/TouchableWithoutFeedback.tsx

```diff
@@ -1,13 +1,12 @@
-// @ts-nocheck TODO(TS) provide types
 import React from 'react';
-import GenericTouchable from './GenericTouchable';
+import GenericTouchable, { GenericTouchableProps } from './GenericTouchable';
 
-const TouchableWithoutFeedback = React.forwardRef((props, ref) => (
-  <GenericTouchable ref={ref} {...props} />
-));
+// TODO: it might give wrong type to users
```

Comment:
TODO.

Reply:
Checked and it shows all the options, so it is ok.